### PR TITLE
Minimise bespoke css, fix markup and update styling

### PR DESCRIPTION
**Pull request by Caleb-Ellis**

### CODE REVIEW

**Comment by kwm14** [comment]
LGTM 👍

File: index.html [hunk 1]
```diff
@@ -67,24 +65,18 @@
...
-                  alt="Snap Store" />
-              </a>
-            </div>
-          </div>
-          <div class="col-4 u-align--center u-vertically--center p-logo">
-            <img class="p-image-shadowed p-logo-certified" src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png?"
-              alt="Certified Kubernetes" />
-          </div>
+
+    <section class="p-strip--dark" style="background-image: url('https://assets.ubuntu.com/v1/8bc629a7-microk8s-suru-background.svg'); background-size: cover;">
```
Comment (anthonydillon) [inline index.html]: `p-strip--image`?

File: index.html [hunk 2]
```diff
@@ -67,24 +65,18 @@
...
+      <div class="row u-equal-height">
+        <div class="col-8">
+          <h1>Kubernetes for workstations and appliances</h1>
+          <h4>A single package of k8s that installs on <s>41</s> 42 flavours of Linux. Made for developers and great for appliances.</h4>
```
Comment (anthonydillon) [inline index.html]: `<p class="p-heading--four">`

File: index.html [hunk 3]
```diff
@@ -228,7 +220,7 @@ <h3 class="p-matrix__title">&nbsp; </h3>
     <section class="p-strip is-bordered" id="quick-start">
       <div class="row">
         <h2>Quick start</h2>
-        <h4>On Ubuntu 16.04 LTS or 18.04 LTS, install the <a href="https://snapcraft.io/microk8s">MicroK8s snap:</a></h4>
+        <h4>On Ubuntu 16.04 LTS or 18.04 LTS, install the <a href="https://snapcraft.io/microk8s">MicroK8s&nbsp;snap:</a></h4>
```
Comment (anthonydillon) [inline index.html]: Paragraph please

**Pull request merged**